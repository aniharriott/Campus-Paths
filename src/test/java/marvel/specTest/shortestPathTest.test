# Find the shortest path between valid characters using shortestPath

LoadGraph short shortestPath.tsv

FindPath short a c