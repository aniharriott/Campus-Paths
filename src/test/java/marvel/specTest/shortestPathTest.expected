# Find the shortest path between valid characters using shortestPath

loaded graph short

path from a to c:
a to c via three